customScripts/expansiontravel.gd

```diff
@@ -2,17 +2,21 @@
 extends Node
 
 func randomfromarray(array):
-	return array[randi() % array.size()]
+	if array.size() > 0:
+		return array[randi() % array.size()]
+	else:
+		return []
 
 func getzonetraveltext(zone,progress):
 	var array = ["Your journey continues peacefully. "]
 	var player = globals.player
 	var party = globals.state.playergroup.duplicate()
 	var teammates = []
-	for i in party:
-		var j = globals.state.findslave(i)
-		if globals.player != j:
-			teammates.append(j)
+	if party.size() > 0:
```

Comment:
This line adds nothing, as for loops do not iterate empty containers nor do they produce errors

Reply:
comment incorporated for new commit